### DISCUSSION TITLE
Skip to Main Content behaviour not expected on PR files page

### DISCUSSION THREAD

**Discussion by Therzok** [community/community#36257] [Accessibility]
When on a PR in the files tab, tabbing and using `Skip to main content` does not move to the PR tabs or the first changed file, but rather the top toolbar that allows you to unsubscribe or mark the notification as done with. That's a bit odd, since I think that's not necessarily content, but an adornment.

There's also the `?` shortcut popover, that is scrollable, but there's no way to move between items in the overlay, and it is scrollable.

**Comment by ossanna16** [comment]
Hey there @Therzok and thank you for the feedback! I'm glad you found our community 🙂

**Comment by cehfisher** [comment]
Hi @Therzok - we wanted to follow up on this reported bug. Is this still an issue for you? If it is resolved, could you please close out this issue? If it is not resolved, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
